Added a Blog link, removed broken blog links, Updated a link.

shivbhadrasinhgohil opened the pull request:
Added a Blog link, removed broken blog links, Updated a link.

aleron75 commented:
Hello @shivbhadrasinhgohil, thanks for your contribution.
My only concern is about Alan Storm's link substitution: the old is specifically Magento oriented, the new that you propose is generical and not Magento oriented so I would not make that change. Ok for the others.

shivbhadrasinhgohil commented:
Yes, you are right. I've updated the link to more the relevant one. I've updated the file. Please check. 
I've chosen the previous link because Alan redirected the old link to this new one but I've found the Magento link.
